### PR TITLE
Implemented "account_put_transaction" method. Added TransactionUtil.t…

### PR DESCRIPTION
**Pull request by zajko**
…s as an implementation for DTO of Transaction entity

### CODE REVIEW

File: src/lib/DeployUtil.ts [hunk 1]
```diff
@@ -137,20 +63,22 @@ export class UniqAddress {
   transferId: BigNumber;
 
   /**
-   * Constructs UniqAddress from the transacting account's `CLPublicKey` and unique transferId.
+   * Builds UniqAddress from the transacting account's `CLPublicKey` and unique transferId.
    * @param publicKey CLPublicKey instance
    * @param transferId BigNumberish value (can be also string representing number). Max U64.
    */
-  constructor(publicKey: CLPublicKey, transferId: BigNumberish) {
```
Comment (zajko) [inline src/lib/DeployUtil.ts:144]: Types used with @jsonObject and TypedJson generally require a no-args constructor

File: e2e/services/CasperServiceByJsonRPC.test.ts [hunk 2]
```diff
@@ -247,6 +258,67 @@ describe('CasperServiceByJsonRPC', () => {
     expect(balanceByUref.eq(balanceByPublicKey)).to.be;
   });
 
+  it.only('should transfer CSPR - account_put_transaction', async () => {
```
Comment (ryoknz) [inline e2e/services/CasperServiceByJsonRPC.test.ts]: could you please remove `.only`?

**Review by hoffmannjan — APPROVED** [review]
tested and works for me! great job!

File: src/lib/Common.ts [hunk 3]
```diff
@@ -0,0 +1,149 @@
...
+import { RuntimeArgs } from './RuntimeArgs';
+
+/**
+ * Deserializes a `Uint8Array` into a hexadecimal string
+ * @param bytes A `Uint8Array` to be deserialized
+ * @returns A base-16 encoded string of the provided byte array
+ */
+export const byteArrayJsonSerializer: (bytes: Uint8Array) => string = (
+  bytes: Uint8Array
+) => {
```
Comment (ryoknz) [inline src/lib/Common.ts:13]: NIT

```suggestion
export const byteArrayJsonSerializer = ( bytes: Uint8Array ): string => {
```

File: src/lib/TransactionUtil.ts [hunk 4]
```diff
@@ -0,0 +1,547 @@
...
+ */
+export const signTransaction = (
+  transaction: Transaction,
+  signingKey: AsymmetricKey
+): Transaction => {
+  const approval = new Approval();
+  if (transaction.Deploy) {
+    signDeploy(transaction.Deploy, signingKey);
+    return transaction;
+  } else if (transaction.Version1) {
```
Comment (ryoknz) [inline src/lib/TransactionUtil.ts:469]: if possible, could we split this into `signTransactionV1`?

File: src/lib/TransactionUtil.ts [hunk 5]
```diff
@@ -0,0 +1,547 @@
...
+}
+
+/**
+ * Builds a `Transaction` object from `TransactionParams`, session logic, and payment logic
+ * @param transactionParam The parameters of the deploy, see [DeployParams](#L1323)
+ * @param session The session logic of the deploy
+ * @param payment The payment logic of the deploy
+ * @returns A new `Transaction` object
+ */
+export function makeV1Transaction(
```
Comment (ryoknz) [inline src/lib/TransactionUtil.ts:517]: NIT: please keep consistency for naming; `TransactionV1` or `V1Transaction`

File: src/services/CasperServiceByJsonRPC.ts [hunk 6]
```diff
@@ -113,6 +116,7 @@ export interface ValidatorsInfoResult extends RpcResult {
 
 /** JSON RPC service for interacting with Casper nodes */
 export class CasperServiceByJsonRPC {
+  oneMegaByte = 1048576;
```
Comment (ryoknz) [inline src/services/CasperServiceByJsonRPC.ts:119]: NIT: Well, I know this comes from the, old code, but for human readability, what about `1 * 1024 * 1024`?

**Review by ryoknz — APPROVED** [review]
This is really great job 👍 , left some NITs can be ignored

**Pull request merged**